products/memgraph/v0.15.0/reference_guide/reading-existing-data.md

```diff
@@ -90,6 +90,23 @@ MATCH (n :Person) WHERE n.age > 20 RETURN n
 Additional examples can be found
 [here](https://neo4j.com/docs/developer-manual/current/cypher/clauses/where/).
 
+#### Regular Expressions
+
+Inside `WHERE` clause, you can use regular expressions for text filtering. To
+use a regular expression, you need to use the `=~` operator.
+
+For example, finding all `Person` nodes which have a name ending with `son`.
+
+```opencypher
+MATCH (n :Person) WHERE ".*son$" RETURN n
```

Comment:
This looks fishy (?)
Isn't there supposed to be an `=~` operator somewhere?

Reply:
Yep, I was just stupid. I thought I copy pasted it from the console, because I tried to see if it runs. I guess I wrote it by hand in the documentation for some reason.